packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enhancements
```

Comment:
If you were using `Popover.Slot` before, it would continue to work.

Some popovers might not render "inline" while they previously did, which could have subtle impacts but I'm not sure it's big enough to consider this a breaking change.

Reply:
Definitely was, previously you could define styles taking advantage of the inline setup, but now CSS selectors need to be rewritten.